lab_1_keywords_tfidf/main.py

```diff
@@ -66,7 +88,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(frequencies, dict) or frequencies == {} \
+            or not isinstance(top, int) or isinstance(top, bool) or top <= 0:
+        return None
+    top_list = []
+    for item in frequencies.items():
```

Comment:
you can split in the loop item to key, value It is a standard way

Reply:
UPD. Now there is no cycle at all